### ISSUE TITLE
[Proposal] Add an example file for importing portfolio

### ISSUE THREAD

**Issue by KristianLentino99** [leonardoarcari/dcapal#39]
It would be nice to have an example file to download to import my portfolio in dcapal. It can be put near the Import portfolio. What do you think?
![image](https://github.com/leonardoarcari/dcapal/assets/60007558/d36f1e62-2e09-4e43-8220-f821baa2a429)

**Comment by corradopetrelli** [comment]
If you want there are the following demo pages:
- https://dcapal.com/demo/60-40
- https://dcapal.com/demo/all-seasons
- https://dcapal.com/demo/mr-rip

It would probably be better to put a page with examples of these 3 portfolios.
What do you think @leonardoarcari ?

**Comment by leonardoarcari** [comment]
Yeah @corradopetrelli is right! There are already those 3 demo portfolios but they are barely advertised.

It would be cool to come up with an home page design that navigates the user to model portfolios if he's not sure where to start.

It shouldn't get in the way of the app presentation, but still should be part of the funnel.

**Comment by corradopetrelli** [comment]
@leonardoarcari 

A naive solution is:
![dcapal com_](https://github.com/leonardoarcari/dcapal/assets/36475130/4654fae9-293b-42be-9600-1ba1fb44324f)
(I didn't know which image to insert so I put the one above)

**Comment by leonardoarcari** [comment]
Yeah probably something like that, but I'd like to work on some cleaner design. Accessing model portfolios shouldn't distract the users from the main workflow.

**Comment by corradopetrelli** [comment]
> Yeah probably something like that, but I'd like to work on some cleaner design. Accessing model portfolios shouldn't distract the users from the main workflow.

No problem it was a naive solution, a possible solution could be to put it in the menu right after "Get started", something like "Try demo".

**Comment by KristianLentino99** [comment]
@corradopetrelli could be a good solution IMHO

**Comment by mattiapitossi** [comment]
What do you think of adding a new section called "explore portfolios" or something like that, maybe in the future we can add a feature where ppl can share their portfolios there. As of now, we can add a few examples to start with.

**Comment by KristianLentino99** [comment]
> We can add a feature where ppl can share their portfolios there
If I understood correctly the project probably won't happen since we don't store anywhere outside the browser the user's portfolios. But make sense to put a section "Demo portfolios" or "Try demo" like @corradopetrelli suggested

**Comment by leonardoarcari** [comment]
> What do you think of adding a new section called "explore portfolios", or something like that, maybe in the future we can add a feature where ppl can share their portfolios there. As of now, we can add a few examples to start with.

Yeah I like this approach @mattiapitossi. A new section will have enough room to put a show-case of demo portfolios while leaving the landing page nice and clean. We could add a short message below the `Allocate your savings` / `Import portfolio` buttons linking to the demo section:

 _Not sure where to start? Check out our model portfolios_ ➡️

> If I understood correctly the project probably won't happen since we don't store anywhere outside the browser the user's portfolios

I certainly value the zero-login/zero-storage experience, but I agree that having a private area could unlock interesting features. Users will trade some privacy for convenience, but non-registered users will still have access to current UX.

**Comment by corradopetrelli** [comment]
> I certainly value the zero-login/zero-storage experience, but I agree that having a private area could unlock interesting features. Users will trade some privacy for convenience, but non-registered users will still have access to current UX.

I love the vision. You could integrate graphs to show the distribution of your portfolio by:
- Brokers
- Currency
- State
- Continent
And so on.

I'm noticing on subreddit /r/ItalyPersonalFinance and all the YouTubers talking about personal finance that the topic of ### diversification is in high demand lately.
In my opinion it could provide added value.

Sorry of OT.

**Comment by leonardoarcari** [comment]
> I'm noticing on subreddit /r/ItalyPersonalFinance and all the YouTubers talking about personal finance that the topic of diversification is in high demand lately

Yeah I noticed! Big hype on investing (betting?) on risky countries in the name of some diversification benefit. Imo, this is just uncompensated risk: you should diversify if there's evidence of decorrelation that brings positive returns. Otherwise you're just reducing returns while increasing your portfolio variance. Brilliant move 😄

**Comment by corradopetrelli** [comment]
> Otherwise you're just reducing returns while increasing your portfolio variance. Brilliant move 😄

Yes of course, it's like to cut off one's nose to spite one's face
